CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+### Changed
+
+- Logic of the fetching process changed: always works, can be restated from a specific block, datumFilter can be changed during work.
+- Reworked control endpoints:
+  - Remove: `POST /control/fetch_blocks`
+  - Remove: `POST /control/cancel_fetch_blocks`
+  - Add: `POST /control/startingBlock`
+  - Add: `POST /control/datumFilter`
+- Reworked WebSocket API:
+  - Remove WS method: `StartFetchBlocks`
+  - Remove WS method: `CancelFetchBlocks`
+  - Add WS method:  `SetStartingBlock`
+  - Add WS method: `SetDatumFilter`
+- Replace configuration file by command line arguments. The config example as CLI: `ogmios-datum-cache --db-connection 'host=localhost port=5432 user=aske dbname=ogmios-datum-cache' --server-port 9999 --server-api 'usr:pwd' --ogmios-address '127.0.0.1' --ogmios-port 1337 --block-slot 44366242 --block-hash d2a4249fe3d0607535daa26caf12a38da2233586bc51e79ed0b3a36170471bf5  --use-latest --block-filter '{"all": [{"hash": "foobar"}, {"any": [{ "address": "addr_abc" },{ "address": "addr_xyz" }]}]}'`
+- Control API requires an authentification token (`--server-api`):
+  - for endpoints by Basic Authorisation
+  - for WebSocket by `token` args.
+
+### Added
+
+- Add `--origin` option to cache datums of all blocks.
+- Add support for Vasil hardfork (require ogmios `>= 5.5.0`).
+- Add `GET /healthcheck` endpoint.
+- Add `GetHealthcheck` WebSocket method.
```

Comment:
I don't have much experience with WS, but do we actually need this method?

Reply:
Clarified in https://github.com/mlabs-haskell/ogmios-datum-cache/pull/93#pullrequestreview-1018282775